Make.config

```diff
@@ -475,10 +475,10 @@ DOTNET_FEED_DIR ?= $(DOTNET_DESTDIR)/nuget-feed
 # We're using preview versions, and there will probably be many of them, so install locally (into builds/downloads) if there's no system version to
 # avoid consuming a lot of disk space (since they're never automatically deleted). The system-dependencies.sh script will install locally as long
 # as there's a TARBALL url.
-DOTNET6_VERSION=6.0.100-alpha.1.21060.3
+DOTNET6_VERSION=6.0.100-alpha.1.21064.27
 DOTNET6_VERSION_BAND=$(firstword $(subst -, ,$(DOTNET6_VERSION)))
-DOTNET6_URL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/6.0.100-alpha.1.21060.3/dotnet-sdk-6.0.100-alpha.1.21060.3-osx-x64.pkg
-DOTNET6_TARBALL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/6.0.100-alpha.1.21060.3/dotnet-sdk-6.0.100-alpha.1.21060.3-osx-x64.tar.gz
+DOTNET6_URL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/6.0.100-alpha.1.21064.27/dotnet-sdk-6.0.100-alpha.1.21064.27-osx-x64.pkg
+DOTNET6_TARBALL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/6.0.100-alpha.1.21064.27/dotnet-sdk-6.0.100-alpha.1.21064.27-osx-x64.tar.gz
```

Comment:
Is it not possible to put `$(DOTNET6_VERSION)` in each of these urls?

Reply:
This is a very very good idea. +1

Reply:
I tried that once, and it breaks in `system-dependencies.sh`, because that file greps Make.config for the url, and ends up with a url with DOTNET6_VERSION in it (instead of the actual version). It's possible to fix it, but I think the fix would be worse than a little bit of copy/paste.